bin/run_ap_pipe.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Set up the stack and necessary ap_packages
+source /software/lsstsw/stack/loadLSST.bash
+setup lsst_distrib
+
+# Set up the cloned and pre-sconsed ap_packages
+L1DBPROTO_DIR=/project/mrawls/l1dbproto
+AP_ASSOC_DIR=/project/mrawls/ap_association
+AP_PIPE_DIR=/project/mrawls/ap_pipe
+setup -k -r $L1DBPROTO_DIR
+setup -k -r $AP_ASSOC_DIR
+setup -k -r $AP_PIPE_DIR
+
+# Set environment variables used to run ap_pipe.py on the command line
+REPO=/project/mrawls/hits2015/
+CALIB=/project/mrawls/hits2015/calib3
+TEMPLATE=/project/mrawls/hits2015/templates
+RERUN=slurm1
+
+# Run ap_pipe in a series of jobs split up by visit and ccdnum
+
+# Note CCDs 2 and 61 are missing, so we process 3-60 inclusive and then 1 and 62
```

Comment:
Again, the filename here says `run_ap_pipe`, but we're now restricted to DECam.  I'm not sure it's worth dealing with the overhead of passing general ccd ranges to this bash script (we don't have the stack tools to use), but maybe it should be called `run_ap_pipe_decam`?